### DISCUSSION TITLE
Equivalent of 'renew now' GUI button

### DISCUSSION THREAD

**Discussion by afmorton** [Venafi/venafips#294] [Q&A] [answered]
I have a requirement to renew a few hundred certificates and in GUI it is easy as single click on the renew now button, but in VenafiPS I have been struggling to find a way to achieve that, I have tried various combinations of New-VdcCertificate or New-VdcObject but failed. Is this possible through VenafiPS. I should add that our users create certificates with nicknames like 'my prod cert' so the cert name is unlikely to be a formed DNS format name, so this could also be a problem as I do get to some interesting error messages such as 'my prod cert' does not match an allowed domain. I did look for the obvious Renew-VdCCertificate but obvioulsy doesn't exist hence looking at combinations of the 'New' commands.

**Comment by gdbarron** [comment, marked as answer]
@afmorton, what you are looking for is `Invoke-VdcCertificateAction -Renew`.  That function has lots of push button actions so worth a look at the help.
